tgui/packages/tgui/interfaces/UploadPanel.js

```diff
@@ -0,0 +1,53 @@
+import { useBackend } from "../backend";
+import { Button, ProgressBar, LabeledList, Box, Section } from "../components";
+import { Window } from "../layouts";
+
+export const UploadPanel = (props, context) => {
+  const { act, data } = useBackend(context);
+  const { selected_target, new_law, id, transmitting, hacked } = data;
+  return (
+    <Window>
+      <Window.Content>
+        <Section title="Silicon Law Upload">
+          <LabeledList>
+            <LabeledList.Item
+              label="Selected Target">
+              <Button
+                disabled={transmitting}
+                selected={selected_target ? true : false}
+                content={selected_target ? selected_target : 'No target selected'}
+                onClick={() => act('target_select')}
+              />
+            </LabeledList.Item>
+            <LabeledList.Item
+              label="Selected Law">
+              <Button
+                disabled={transmitting}
+                selected={new_law ? true : false}
+                content={new_law ? new_law : 'No module installed'}
+                onClick={() => act('law_select')}
+              />
+            </LabeledList.Item>
+            <LabeledList.Item
+              label="Authorization">
+              <Button
+                selected={id ? true : false}
+                content={id ? id : (hacked ? '$@!ERR0R!@#' : 'No ID card inserted')}
```

Comment:
```suggestion
                content={id || (hacked ? '$@!ERR0R!@#' : 'No ID card inserted')}
```
Чуть меньше тернарников.